### PR TITLE
Seed data refactor

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: ui_automation_tests/fixtures/core.py [hunk 1]
```diff
@@ -90,3 +90,46 @@ def clear_down(context, api_url):
 @fixture(scope="session")
 def new_cases_queue_id():
     return "00000000-0000-0000-0000-000000000001"
+
+
+@fixture(scope="session")
+def exporter_info(request):
+    exporter_sso_email = env('TEST_EXPORTER_SSO_EMAIL')
```
Comment (phlptt) [inline ui_automation_tests/fixtures/core.py]: why is any exporter stuff needed?

File: ui_automation_tests/fixtures/core.py [hunk 2]
```diff
@@ -90,3 +90,46 @@ def clear_down(context, api_url):
...
+    first_name = 'Test'
```
Comment (phlptt) [inline ui_automation_tests/fixtures/core.py]: maybe store this as its shared lower down first name and last name Test Lite is repeated

File: ui_automation_tests/helpers/request_data.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
+org_name = 'Test Org'
+org_name_for_switching_organisations = 'Octopus Systems'
+logging = True
+case_note_text = 'I Am Easy to Find'
+ecju_query_text = 'This is a question, please answer'
+first_name = 'Test'
```
Comment (phlptt) [inline ui_automation_tests/helpers/request_data.py]: repeated

File: ui_automation_tests/fixtures/core.py [hunk 4]
```diff
@@ -90,3 +90,46 @@ def clear_down(context, api_url):
...
+    }
+
+
+@fixture(scope="session")
+def s3_key(request):
+    s3_key = env('TEST_S3_KEY')
+    return s3_key
+
+
+@fixture(scope="session")
```
Comment (jfaracik) [inline ui_automation_tests/fixtures/core.py]: Single quotes please!

File: ui_automation_tests/helpers/request_data.py [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+            'user': {
+                'first_name': exporter_user['first_name'],
+                'last_name': exporter_user['last_name'],
+                'email': exporter_user['email']
+            },
+            'site': {
+                'name': 'Headquarters',
+                'address': {
+                    'address_line_1': '42 Question Road',
+                    'postcode': 'Islington',
```
Comment (jfaracik) [inline ui_automation_tests/helpers/request_data.py]: Is this going to be moved to the submodule/is there a plan to split this out into functions? eg create_site(**kwargs)...

Comment (BenAndrew-Kainos) [reply ui_automation_tests/helpers/request_data.py]: exactly that!

File: ui_automation_tests/fixtures/core.py (hunk 4 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/fixtures/core.py]: done

File: ui_automation_tests/fixtures/core.py [hunk 6]
```diff
@@ -90,3 +90,46 @@ def clear_down(context, api_url):
...
+        'email': exporter_sso_email,
+        'first_name': first_name,
+        'last_name': last_name
+    }
+
+
+@fixture(scope="session")
+def internal_info(request):
+    gov_user_email = env('TEST_SSO_EMAIL')
+    gov_user_first_name = env('TEST_SSO_NAME').split(' ')[0]
```
Comment (stephanieboyd) [inline ui_automation_tests/fixtures/core.py]: Why not split once? 

`first, second = env('TEST_SSO_NAME').split(' ')`

Comment (BenAndrew-Kainos) [reply ui_automation_tests/fixtures/core.py]: good point. Spotted that and forgot about it

**Pull request merged**